posix/basic-unixint.lisp

```diff
@@ -298,6 +298,11 @@
   (ctype blksize "long")
   (ctype blkcnt "long"))
 
+#-windows
```

Comment:
This conditional is unnecessary.

Reply:
Ok, I have no opportunity to test it on Windows now, but I've just removed the condition.